test/images/n-way-http/VERSION

```diff
@@ -0,0 +1 @@
+0.1
```

Comment:
The version in the gcr is completely different and I see version is 1.0. Let me put 1.1 here.